server/src/store.js

```diff
@@ -75,10 +80,40 @@ const handComment = {
 };
 const comments = [foxComment1, foxComment2, handComment];
 
+AWS.config.update({ region: "us-east-2" });
+const dynamoDb = new AWS.DynamoDB.DocumentClient();
+
 export function getBits() {
   return bits;
 }
 
+export function createBit(bit) {
+  const params = {
+      TableName: 'posts',
+      Item: {
+        postId: uuid.v1(),
+        author: {
+          name: jsStringEscape(bit.author.name),
+          personId: jsStringEscape(bit.author.person_id)
+        },
+        dateCreated: new Date().getTime(),
+        upvotes: 0,
+        downvotes: 0,
+        title: jsStringEscape(bit.title),
+        content: jsStringEscape(bit.content),
+        ...(bit.tags ? { tags: jsStringEscape(bit.tags.join()) } : {}),
+        ...(bit.stages ? { stages: jsStringEscape(bit.stages.join()) } : {}),
+        ...(bit.mainChars ? { main_chars: jsStringEscape(bit.main_chars.join()) } : {}),
+        ...(bit.vsChars ? { vs_chars: jsStringEscape(bit.vsCharss.join()) } : {})

```

Comment:
This syntax is needlessly pathological. I'd rather something like

```javascript
tags: bit.tags ? jsStringEscape(bit.tags.join()) : undefined
```

or something like that.

Reply:
Dynamo won't let you commit undefined properties though :-/

Reply:
Ah I see. I'd rather logic for sanitizing a user-given bit and creating a db-ready model somewhere outside of this file. And I would also prefer

```javascript
if (bit.tags) {
  item['tags'] = jsStringEscape(bit.tags.join());
}
```

Actually maybe even better would be my proposed syntax above, and then something like:

```javascript
function stripUndefinedFields(obj) {
  for (const prop in obj) {
    if (prop.hasOwnProperty(key) && prop[key] === undefined) {
      delete obj[prop];
    }
  }
}
```